src/main/java/nextstep/subway/domain/Distance.java

```diff
@@ -30,8 +30,12 @@ public Integer getValue() {
         return value;
     }
 
-    public Distance minus(Integer value) {
-        return new Distance(this.value - value);
+    public Distance minus(Distance distance) {
+        return new Distance(this.value - distance.getValue());
```

Comment:
```suggestion
        return new Distance(this.value - distance.value);
```

Distance를 전달 받았으니 `getter`를 사용안할수 있겠네요! :+1: